web/src/components/timeMachine/timesheets/AddNewTimeRecord.vue

```diff
@@ -143,10 +143,11 @@
                 <v-flex xs12 sm5>
                   <v-text-field
                     label="Hours"
+                    
                     type="number"
                     :rules="requiredRule"
                     :min="0"
-                    step="any"
+                    step="0.25"
```

Comment:
This doesn't work, step has to be 0.1 or completely removed.

Reply:
My suggested course of action: 

- Remove step entirely OR revert it to step="0.1"
- Add a :rules="hoursRule
- Create an hoursRule function that parses the number and makes sure it's divisible by 0.25.